### PR TITLE
Add notes about garbage collection

### LINKED ISSUE #66: HOWTO run garbage collection in the docker?
I `bash`ed into the docker, stopped the server, ran `seaf-gc.sh --dry-run` and it went through some libraries but then terminated the docker.

Have I missed something?

### PR DESCRIPTION
**Pull request by yatesco**
Fixes #66

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +126,12 @@ docker run -d --name seafile \
 
 If you are one of the early users who use the `launcher` script, you should refer to [upgrade from old format](https://github.com/haiwen/seafile-docker/upgrade_from_old_format.md) document.
 
+### Garbage Collection
+
+When files are deleted, the blocks comprising those files are marked as unused but not immediately removed. To remove them, Seafile requires a ['garbage collection'](https://manual.seafile.com/maintain/seafile_gc.html) process to be run.
```
Comment (shoeper) [inline README.md]: Technically they are not marked as deleted. The garbage collector tries to find out whether they are still used or not. It is only guaranteed that no required blocks are removed. It is not guaranteed that all blocks that could be removed are actually getting removed (some might remain and be deleted in another run).

File: README.md [hunk 2]
```diff
@@ -126,6 +126,12 @@ docker run -d --name seafile \
...
+
+The required scripts can be found in the `/scripts` folder of the docker container. To perform garbage collection, simply run `docker exec seafile /scripts/gc.sh`. For the community edition, this process will stop the seafile server, but it is a relatively quick process and the seafile server will start automatically once the process has finished. The Professional supports an online garbage collection.
```
Comment (shoeper) [inline README.md]: This is correct, but gc.sh needs to check whether it is CE or Pro and only stop when it is CE.

@zMingGit @freeplant

**Comment by shoeper** [comment]
@freeplant

**Comment by freeplant** [comment]
Thank you!

**Comment by yatesco** [comment]
No worries at all, and thank YOU (all) for Seafile

**Pull request merged**